language/compiler/src/main.rs

```diff
@@ -59,6 +62,14 @@ fn do_verify_module(module: CompiledModule, dependencies: &[CompiledModule]) ->
     module
 }
 
+fn do_verify_script(script: CompiledScript, dependencies: &[CompiledModule]) -> CompiledScript {
```

Comment:
Thoughts on a slightly different signature:
```
fn do_verify_script(script: &CompiledScipt, deps: ...) -> ()
```
Feels a bit weird to take this as owned just to pass it back